src/statisticBlock.js

```diff
@@ -345,9 +353,19 @@ class StatisticBlock extends Component {
                 size: dimLabelSize,
                 text: dimensionLabel
               };
-
+              // console.log(dimensionIndex);
+              
+              // console.log(labelOptions);
+              // let isSelected ;
+              // if(arrayOfValues.indexOf(dimensionIndex) > -1){
+              //   isSelected = true
+              // }else{
+              //   isSelected= false;
+              // }
               return (
                 <DimensionEntry
```

Comment:
Would be nice if you could remove the key={Math.random()} property here (hack should no longer be needed)